routing/route.hpp

```diff
@@ -32,6 +33,52 @@ namespace routing
 using SubrouteUid = uint64_t;
 SubrouteUid constexpr kInvalidSubrouteId = std::numeric_limits<uint64_t>::max();
 
+/// \brief The route is composed of one or several subroutes. Every subroute is composed of segments.
+/// For every Segment is kept some attributes in the structure SegmentInfo.
+class RouteSegment final
+{
+public:
+  RouteSegment(Segment const & segment, turns::TurnItem const & turn, Junction const & junction,
+               std::string const & street, double distFromBeginningMeters, double distFromBeginningMerc,
+               double timeFromBeginningS, traffic::SpeedGroup traffic)
+    : m_segment(segment)
+    , m_turn(turn)
+    , m_junction(junction)
+    , m_street(street)
+    , m_distFromBeginningMeters(distFromBeginningMeters)
+    , m_distFromBeginningMerc(distFromBeginningMerc)
+    , m_timeFromBeginningS(timeFromBeginningS)
+    , m_traffic(traffic)
+  {
+  }
+
+  Segment const & GetSegment() const { return m_segment; }
+  turns::TurnItem const & GetTurn() const { return m_turn; }
+  Junction const & GetJunction() const { return m_junction; }
+  std::string const & GetStreet() const { return m_street; }
+  double GetDistFromBeginningMeters() const { return m_distFromBeginningMeters; }
+  double GetDistFromBeginningMerc() const { return m_distFromBeginningMerc; }
+  double GetTimeFromBeginningS() const { return m_timeFromBeginningS; }
+  traffic::SpeedGroup GetTraffic() const { return m_traffic; }
+
+private:
+  Segment m_segment;
+  /// Turn (maneuver) information for the turn next to the |m_segment| if any.
+  /// If not |m_turn::m_turn| is equal to TurnDirection::NoTurn.
+  turns::TurnItem m_turn;
+  /// The furthest point of the segment from the beginning of the route along the route.
+  Junction m_junction;
+  /// Street name of |m_segment| if any. Otherwise |m_street| is empty.
+  std::string m_street;
+  /// Distance from the route beginning to the farthest end of |m_segment| in meters.
+  double m_distFromBeginningMeters = 0.0;
+  /// Distance from the route beginning to the farthest end of |m_segment| in mercator.
```

Comment:
I do not understand what does "distance in mercator" means. Mercator is a way to describe coordinates on the sphere, so from my point of view "distance in mercator" is sounds like "distance in lat-lons" or "distance in kilos". Could you please explain how this field should be used?